src/modules/MethodsAPI.js

```diff
@@ -0,0 +1,30 @@
+import htmldisplay from './htmldisplay.js';
+
+const baseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/ygzvsig2RDHlS9A3YwuH/scores';
+
+export const PostScore = async (newPlayer) => {
+  console.log('myPlayer', newPlayer);
+  const post = await fetch(baseUrl, {
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+    body: JSON.stringify({ user: newPlayer.user, score: newPlayer.score }),
+  });
+  console.log('Getpost', post);
+
+  const response = await post.json();
+  return response;
+};
```

Comment:
- [ ] Please consider using the correct naming convention for this variable, at the moment, you're using `pascal case` which is most widely used for classes. The naming convention for variables and functions should be in `camel case`, so please apply this change! 🤓 🔧 

- [ ] Also, according to the [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md), please avoid committing console.log calls: 🤓 🔧 

![image](https://user-images.githubusercontent.com/77069011/217978348-3538e48d-6d00-4e4c-9c95-109ab42264ff.png)

Reply:
It is fixed